packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md

```diff
@@ -2,27 +2,28 @@ Display the user info by updating the `showUserInfo` function of `ContentView`:
 
```

Comment:
I can't comment the top line for some reason. For consistency, should you have `ContentView.swift` rather than just `ContentView` or should it say "the `ContentView` struct"?

Reply:
Good catch. Changed to the file name. In the next commit.